### PR TITLE
Bulk text formatting updates to the clusterawareupdating module

### PR DESCRIPTION
**Pull request by robinharwood**
Bulk update to the text formatting for the clusterawareupdating module before applying further updates.

### CODE REVIEW

File: docset/winserver2022-ps/clusterawareupdating/Invoke-CauRun.md [hunk 1]
```diff
@@ -520,13 +522,13 @@ Accept wildcard characters: False
...
 succeed if the underlying clustered space is in degraded condition.
 
-If **Suspend-ClusterNode** fails with ERROR_CLUSTER_SPACE_DEGRADED error, CAU will keep retrying for
-*SuspendClusterNodeTimeoutMinutes* or suspend the call if the command succeeds.
+If `Suspend-ClusterNode` fails with ERROR_CLUSTER_SPACE_DEGRADED error, CAU will keep retrying for
+**SuspendClusterNodeTimeoutMinutes** or suspend the call if the command succeeds.
 
-The retries for this error do not count towards the *MaxRetriesPerNode* parameter set by the user
+The retries for this error don't count towards the **MaxRetriesPerNode** parameter set by the user
 parameter set by the user.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/clusterawareupdating/Invoke-CauRun.md:532]:
```suggestion
The retries for this error don't count towards the **MaxRetriesPerNode** parameter set by the user.
```

File: docset/winserver2022-ps/clusterawareupdating/Invoke-CauScan.md [hunk 2]
```diff
@@ -177,11 +177,11 @@ Accept wildcard characters: False
 
 ### -RunPluginsSerially
 Indicates that CAU scans each cluster node for applicable updates and stages the updates for each
-plug-in in the plug-in order passed into the *CauPluginName* parameter then multiple plug-ins are
+plug-in in the plug-in order passed into the **CauPluginName** parameter then multiple plug-ins are
 used during a scan for updates
 
 By default, CAU scans and stages the applicable updates for all plug-ins in parallel.
-This parameter is valid only when multiple plug-ins are specified in *CauPluginName* parameter.
+This parameter is valid only when multiple plug-ins are specified in **CauPluginName** parameter.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/clusterawareupdating/Invoke-CauScan.md:184]:
```suggestion
This parameter is valid only when multiple plug-ins are specified in the **CauPluginName** parameter.
```

File: docset/winserver2022-ps/clusterawareupdating/Set-CauClusterRole.md [hunk 3]
```diff
@@ -501,7 +504,7 @@ By default, CAU scans and stages the applicable updates for all plug-ins in para
 the configuration of this parameter, CAU installs the applicable updates for each plug-in
 sequentially.
 
-The parameter is valid only when multiple plug-ins are specified in *CauPluginName* parameter. If a
+The parameter is valid only when multiple plug-ins are specified in **CauPluginName** parameter. If a
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/clusterawareupdating/Set-CauClusterRole.md:507]:
```suggestion
The parameter is valid only when multiple plug-ins are specified in the **CauPluginName** parameter. If a
```

File: docset/winserver2022-ps/clusterawareupdating/Invoke-CauRun.md [hunk 4]
```diff
@@ -73,12 +75,12 @@ PS C:\> Invoke-CauRun -ClusterName CONTOSO-FC1 -CauPluginName Microsoft.WindowsU
 This command performs a scan and a full updating run on the cluster named CONTOSO-FC1. This cmdlet
 uses the **Microsoft.WindowsUpdatePlugin** plug-in with the default configuration, and the
 **Microsoft.HotfixPlugin** plug-in using the hotfix root folder \\\\CauHotfixSrv\shareName and the
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/clusterawareupdating/Invoke-CauRun.md:77]:
```suggestion
**Microsoft.HotfixPlugin** plug-in using the hotfix root folder `\\CauHotfixSrv\shareName` and the
```

File: docset/winserver2022-ps/clusterawareupdating/Set-CauClusterRole.md [hunk 5]
```diff
@@ -597,10 +600,10 @@ Accept wildcard characters: False
 Specifies the maximum amount of time CAU should wait for the Suspend-ClusterNodecmdlet to succeed if
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/clusterawareupdating/Set-CauClusterRole.md:600]:
```suggestion
Specifies the maximum amount of time CAU should wait for the `Suspend-ClusterNode` cmdlet to succeed if
```

File: docset/winserver2022-ps/clusterawareupdating/Add-CauClusterRole.md [hunk 6]
```diff
@@ -88,12 +90,12 @@ PS C:\> Add-CauClusterRole -ClusterName "CONTOSO-FC1" -CauPluginName Microsoft.W
 
 This command adds the CAU clustered role, using a default name, on the cluster called CONTOSO-FC1.
 The CAU clustered role is configured to perform updates using the **Microsoft.WindowsUpdatePlugin**
-plug-in with the optional *IncludeRecommendedUpdates* parameter set to True, and using the
+plug-in with the optional **IncludeRecommendedUpdates** parameter set to True, and using the
 **Microsoft.HotfixPlugin plug-in** using the hotfix root folder \\\\CauHotfixSrv\shareName and the
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/clusterawareupdating/Add-CauClusterRole.md:94]:
```suggestion
**Microsoft.HotfixPlugin plug-in** using the hotfix root folder `\\CauHotfixSrv\shareName` and the
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @robinharwood, kindly commit the few additional suggested changes. Many thanks.

**Comment by robinharwood** [comment]
Hi @sdwheeler / @michaeltlombardi,

@JohanFreelancer9 kindly completed a copy review of this PR. I have committed his suggested changes and I believe this is ready for #sign-off. If you would be so kind as to review it and let me know.

Thanks,

Robin

**Review by JohanFreelancer9 — APPROVED** [review]
@robinharwood, copy looks good, thanks.

**Pull request merged**